docs/install/binary.mdx

```diff
@@ -195,6 +195,25 @@ After adding the repository, install Ghostty:
 rpm-ostree update --install ghostty
 ```
 
+
+### OpenSUSE
+
+Ghostty is available for openSUSE Tumbleweed:
+
+```sh
+zypper addrepo https://download.opensuse.org/repositories/home:avindra/openSUSE_Tumbleweed/home:avindra.repo
```

Comment:
This has moved to `X11:terminals`  (https://build.opensuse.org/package/show/X11:terminals/ghostty).

I'll raise a PR with the official package when it's ready (should be ready within ~2-3 days).